include/eve/concept/generator.hpp

```diff
@@ -21,12 +21,34 @@ namespace eve
   //! @concept generator
   //! The concept `generator<Constant>` is satisfied if `Constant` is built as a eve::constant_callable.
   //!
+  //! @tparam G Type of the generator to check.
+  //!
   //! @groupheader{Examples}
-  //!   - `eve::one`
+  //!   - `eve::one_t`
```

Comment:
I undrestand where you coming from but this is not a helpful doc.

eve::one is what people should be using. one_t is almost never used.

@jfalcou  help me here.

Reply:
Yeah, I asked @SadiinsoSnowfall to change it as technically the concept works on type not value.
We can change it back no problem.